src/command_modules/azure-cli-container/azure/cli/command_modules/container/custom.py

```diff
@@ -114,7 +119,8 @@ def create_container(client,
                             image_registry_credentials=image_registry_credentials,
                             volumes=volumes or None)
 
-    return client.create_or_update(resource_group_name, name, cgroup)
+    raw_response = client.create_or_update(resource_group_name, name, cgroup, raw=True)
+    return raw_response.output
```

Comment:
What is the reason for this change?

Reply:
We are preserving the current CLI output for create/update container group which is immediately return. Since this Sdk introduced LRO, we want to wait until the next release to have functionality change.